src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -40,6 +40,8 @@
 
 ACCEPTED_METRIC_TYPES = {"s", "c", "d"}  # set, counter, distribution
 
+SENTRY_RECEIVED_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
Can you use a float instead of introducing this format? Otherwise this needs to be in both Sentry + Snuba. It is also subtly different from https://github.com/getsentry/snuba/blob/578bf7ad56adf4c951b57caaabd9518af989e020/snuba/settings/__init__.py#L212 which can cause confusion.

Reply:
updated and updated tests accordinly